docs/android/emulator/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Android Emulator Setup"
+description: "The Android Emulator can be run in a variety of configurations to simulate different devices. This article explains how to prepare the Android Emulator for testing your .NET MAUI app."
+ms.date: 12/02/2021
+---
+
+# Android emulator setup
+
+The Android Emulator can be run in a variety of configurations to simulate different devices. Each configuration is called a _virtual device_. When you deploy and test your app on the emulator, you select a pre-configured or custom virtual device that simulates a physical Android device such as a Nexus or Pixel phone.
+
+The following articles are available to help manage and setup your Android emulator:
```

Comment:
The debugging article is missing from the list, but seems to be mentioned in the final sentence in the doc.

Reply:
So to be honest I wouldn't have migrated the index.md article. For me, it adds no value. The Xamarin docs are full of such articles (a hangover from before the docs were on docs.microsoft.com), and in 95% of cases I've been deleting them as I migrate content. I just assumed that as you'd migrated it you saw some value in it.